code/packages/web/src/pages/App.tsx

```diff
@@ -0,0 +1,22 @@
+// App.tsx
+import React from 'react';
+import { BrowserRouter as Router, Routes, Route } from 'react-router-dom';
+import LandingPage from './components/LandingPage';
+import LoginPage from './components/LoginPage';
+import SignupPage from './components/SignuPage';
+import ForgotPasswordPage from './components/ForgotPasswordPage';
+
+function App() {
+  return (
+    <Router>
```

Comment:
This will not work unless you wrap `<App />` in `main.tsx` with RouterProvider

Please read React Router documentation at: [React Router](https://reactrouter.com/en/main)